sphinx_gallery/directives.py

```diff
@@ -190,6 +176,28 @@ def _parse_srcset(st):
     return srcset
 
 
+def _copy_images(self, node):
```

Comment:
Can you move this back down to minimize the `diff`? It helps with `git blame`

Reply:
Yeah diff is ugly, I'll revert.